.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+sudo: false
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 7
+  - hhvm
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
@pleckey 

:information_desk_person: To really make use of caching, one should create a GitHub token and use it to authenticate with the GitHub API:

1. Acquire a GitHub authentication token (go to https://github.com/settings/tokens/new)
2. Install the `travis` command line tool (see https://github.com/travis-ci/travis.rb#installation)
3. Encrypt the GitHub authentication token

    ```
$ travis encrypt GITHUB_TOKEN=foobarbaz
    ```
4. Add the encrypted token to `.travis.yml`

    ```yml
    env:
      global:
        - secure: "..."
    ```
5. Use token to authenticate against GitHub API before installing dependencies

    ```yml
    before_install:  
      - composer config github-oauth.github.com $GITHUB_TOKEN
    ```